src/main/resources/xslt/outputs/odt/models.xsl

```diff
@@ -1,544 +1,641 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same comments as /input/ddi/source-fixed.xsl one. As this file is only for odt output transformation, as a commit pattern, these kind of "delete all, paste all" commit should be avoided.